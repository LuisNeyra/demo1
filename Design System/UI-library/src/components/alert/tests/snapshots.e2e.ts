import { snapIt } from '../../../test';

const component = 'alert';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('without parameters', '<blaze-alert>test this!</blaze-alert>');
    snap('correct type', '<blaze-alert type="brand">test this!</blaze-alert>');
  });
});
