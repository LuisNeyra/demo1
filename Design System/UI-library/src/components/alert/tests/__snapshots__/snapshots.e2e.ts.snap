// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`alert renders correct type 1`] = `"<blaze-alert type=\\"brand\\" class=\\"hydrated\\"><!----><div role=\\"alert\\" hidden=\\"\\" class=\\"c-alert c-alert--brand u-display-none\\">test this!</div></blaze-alert>"`;

exports[`alert renders without parameters 1`] = `"<blaze-alert class=\\"hydrated\\"><!----><div role=\\"alert\\" hidden=\\"\\" class=\\"c-alert u-display-none\\">test this!</div></blaze-alert>"`;
